internal/venice-consumer/src/main/java/com/linkedin/venice/pubsub/api/PubSubTopicType.java

```diff
@@ -0,0 +1,21 @@
+package com.linkedin.venice.pubsub.api;
+
+import com.linkedin.venice.exceptions.VeniceException;
+import com.linkedin.venice.meta.Version;
+
+
+public enum PubSubTopicType {
+  VERSION_TOPIC, REALTIME_TOPIC, REPROCESSING_TOPIC;
```

Comment:
If this is going to be stored somewhere, consider adding values for enums. Refer to `enum IncrementalPushPolicy` for a sample.

Reply:
At this moment, I do not think we will store this somewhere at this moment. If we need to store it, we can change this afterwards.

Reply:
ok